config.go

```diff
@@ -4,7 +4,7 @@ import (
 	"io/ioutil"
 	"strings"
 
-	"github.com/drone/drone/pkg/build/log"
+	"github.com/drone/drone/shared/build/log"
```

Comment:
ah ok I'll update it

On Friday, February 20, 2015, Michael Crosby <notifications@github.com>
wrote:

> In config.go
> <https://github.com/crosbymichael/slex/pull/4#discussion_r25086620>:
>
> > @@ -4,7 +4,7 @@ import (
> >  	"io/ioutil"
> >  	"strings"
> >
> > -	"github.com/drone/drone/pkg/build/log"
> > +	"github.com/drone/drone/shared/build/log"
>
> oh, this is just a bad go import, it should be logrus as the logger
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crosbymichael/slex/pull/4/files#r25086620>.
>